TF_Service_dotNet/TouchFree_Service/Program.cs

```diff
@@ -15,6 +15,11 @@ static void Main(string[] args)
         {
 #if !DEBUG
             var loggingFileDirectory = RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ? ConfigFileUtils.ConfigFileDirectory : "";
+
+            if (loggingFileDirectory != "") {
+                Directory.CreateDirectory(loggingFileDirectory);
```

Comment:
Do we need to check whether the folder already exists or does this handle that? I can check but just wondering if you'd already tested it

Reply:
I ended up putting the check in because it failed the integration tests, as you can't give CreateDirectory an empty string. 👍